src/app/courses/components/course-card/course-card.component.scss

```diff
@@ -0,0 +1,82 @@
+.card-container {
+  margin: 0 85px;
+  padding: 24px 25px;
+  height: 140px;
+  border-radius: 8px;
+  background-color: #ffffff;
+  margin-bottom: 24px;
+
+  .title-container {
+    display: flex;
+    justify-content: space-between;
+
+    .title {
+      font-size: 24px;
+      font-weight: 600;
+      display: inline-block;
+      font-stretch: normal;
+      font-style: normal;
+      line-height: 1.25;
+      color: #2c2f3c;
+    }
+
+    span,
+    .course-duration-wrapper {
+      display: flex;
+      justify-content: flex-end;
+
+      span {
+        color: #b9bdce;
+        margin-right: 10px;
+      }
+    }
+  }
+
+  .course-info {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+
+    .description {
+      font-size: 14px;
+      margin: 0;
+      letter-spacing: normal;
+      color: #525462;
+    }
+
+    .btn-container {
+      display: flex;
+      height: 30px;
+      align-self: center;
+      justify-content: flex-end;
+
+      .btn {
+        display: flex;
```

Comment:
Could you please make left and right padding of the buttons visually equal?

Reply:
fixed thanks